### DISCUSSION TITLE
Challenge aceppted! Gives FBW a little love

### DISCUSSION THREAD

**Discussion by Daboj1337** [flybywiresim/aircraft#6485] [Show and tell]
Takes it with humor despite the disasters in the flight plan drawing  

<img width="673" alt="Unbenannt" src="https://user-images.githubusercontent.com/76589309/148226401-9cef4980-0d5d-432c-a197-7d222ef867c4.PNG">

**Comment by andrejetcat** [comment]
CUTE!
I´m smiling every time, when I see such strange drawings for approaches and departures...because that indicated, HOW MUCH WORK it it to make it right! And every time I´m shure that we will see accurate drawings one time!

So THANK YOU again FBW for all your hard work!
